fixtures/artifactor_plugin.py

```diff
@@ -156,6 +156,7 @@ def pytest_unconfigure():
     art_client.fire_hook('teardown_merkyl', ip=appliance_ip_address)
     if not SLAVEID:
         art_client.terminate()
+        proc.wait()
```

Comment:
Not long, this was added to address a race condition that was clipping a few tests off the end of the report. Plus, we already write something to the console ('collecting artifacts')